DigitalOffice/Tests/CompanyServiceTests/CompanyServiceUnitTests.csproj

```diff
@@ -11,9 +11,15 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.EntityFrameworkCore" Version="3.1.6" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.InMemory" Version="3.1.6" />
     <PackageReference Include="nunit" Version="3.12.0" />
```

Comment:
If we use only InMemory database, there's no need for the first package.

Reply:
fixed